Library/Homebrew/cask/lib/hbc/cli/internal_stanza.rb

```diff
@@ -43,6 +43,10 @@ def initialize(*)
         @stanza = args.shift.to_sym
 
         @format = :to_yaml if yaml?
+
+        return if DSL::DSL_METHODS.include?(stanza)
+        raise ArgumentError,
+          "Unknown/unsupported stanza: '#{stanza}' Check cask reference for supported stanzas."
```

Comment:
This is an example of an excellent error message. Not only does it clearly state what the problem is, it also tells the user exactly how to fix it. 👍 

For longer messages like this one, I suggest to use the `\` line continuation operator to keep your code within the 80-character boundary:

```
raise ArgumentError,
  "Unknown/unsupported stanza: '#{stanza}'. " \
  "Check cask reference for supported stanzas."
```